cap/01-categorie.tex

```diff
@@ -1,9 +1,162 @@
+  \item Esiste un unico morfismo da $x$ a $y$ (e uno da $y$ a $x$) se e solo se $x\sim y$. 
+ \end{itemize}
+ Ad ogni oggetto $x$, l'identità è l'unico morfismo $x\to x$ dato dal fatto che la relazione è riflessiva ($x\sim x$). 
+ La composizione è data dalla transitività: se abbiamo morfismi $x\to y$ e $y\to z$ significa, in particolare, che $x\sim y$ e $y\sim z$. Per transitività, $x\sim z$, e quindi c'è un unico morfismo $x\to z$, che possiamo prendere come composizione. 
+ Gli assiomi di unitalità e associatività sono automaticamente soddisfatti: per esempio, dati $w\sim x\sim y\sim z$, per transitività abbiamo che $w\sim z$, e questo dà un \emph{unico} morfismo $w\to z$, non importa se lo otteniamo componendo prima $w\sim x$ e $x\sim y$ o prima $x\sim y$ e $y\sim z$ -- il risultato è lo stesso per unicità. 
+\end{example}
+
+Nell'esempio sopra vediamo che \emph{gli oggetti non sono insiemi con strutture ulteriori}. 
+Notiamo anche che la proprietà di \emph{simmetria} delle relazioni di equivalenza non serve per avere una categoria. Consideriamo i seguenti esempi.
+
+\begin{examples}
+ Una relazione d'ordine $(X,\le)$ si può vedere come una categoria, ancora una volta con un unico morfismo $x\to y$ se e solo se $x\le y$. Come nel caso delle relazioni di equivalenza, le identità sono date dalla proprietà riflessiva e la composizione dalla proprietà transitiva. 
+ 
+ Più in generale, un \emph{preordine} è una relazione riflessiva e transitiva, ma non necessariamente simmetrica o antisimmetrica. Ogni preordine si può vedere come una categoria.
+\end{examples}
+
+
+\begin{example}
+ Un monoide si può vedere come una categoria con un solo oggetto -- più precisamente, una categoria \emph{localmente piccola} con un solo oggetto. 
+ Vediamo come. Dato un monoide $M$, definiamo la seguente categoria, che indichiamo con $\ctB M$.\footnote{A volte la categoria definita in questo modo si indica con $\ctB M$, a volte semplicemente con $M$.}
+ \begin{itemize}
+  \item La categoria $\ctB M$ ha un unico oggetto, che indichiamo con un punto, $\bullet$;
+  \item La categoria $\ctB M$ ha un morfismo $m:\bullet\to\bullet$ per ogni elemento $m\in M$;
+  \item L'identità dell'unico oggetto $m$ e il morfismo definito dall'elemento neutro $e\in M$;
+  \item La composizione di morfismi è data dal prodotto in $M$.
+ \end{itemize}
+ In un monoide, a differenza di una categoria generica, possiamo sempre comporre due morfismi $m$ e $n$: questo è garantito dal fatto che siccome c'è un solo oggetto, il dominio e il codominio di $m$ ed $n$ sono necessariamente uguali. 
+ 
+ Viceversa, data una categoria localmente piccola con un solo oggetto (indichiamolo ancora con questo simbolo, $\bullet$), l'insieme di morfismi $\bullet\to\bullet$ ha una struttura di monoide con l'elemento neutro dato dall'identità, e il prodotto dato dalla composizione.
+ 
+ L'unitalità e l'associatività del monoide e della categoria si corrispondono.
+\end{example}
```

Comment:
What if questo esempio venisse trasformato in uno statement preciso (definiamo la "classe" delle categorie localmente piccole con un singolo oggetto; definiamo la classe dei monoidi; ci sono due corrispondenze in direzioni opposte, definite da... che sono mutuamente inverse. Se in questa corrispondenza proviamo a scrivere cosa significa essere un omomorfismo di monoidi, vediamo che ottieniamo una corrispondenza del genere a livello delle categorie con un solo oggetto: bla bla bla. Questo dà un modo secondo me molto efficace di introdurre la nozione di funtore (la introduco circa così quando insegno).